v0/destinations/pinterest_tag/utils.js

```diff
@@ -110,7 +110,7 @@ const processCommonPayload = message => {
     !VALID_ACTION_SOURCES.includes(presentActionSource.toLowerCase())
   ) {
     throw new CustomError(
-      "Action source must be one of 'app_android', 'app_ios', 'web', 'offline' ",
+      "Action source must be one of 'app_android', 'app_ios', 'web', 'offline'",
```

Comment:
```suggestion
      `Action source must be one of ${VALID_ACTION_SOURCES}`,
```

Reply:
Action source must be one of ${VALID_ACTION_SOURCES} this will output
`Action source must be one of app_android, app_ios, web, offline`

Reply:
Yes @koladilip

Reply:
I tried to reuse but it is getting over complicated, if add library binding for this then we can use it later.

Reply:
@saikumarrs if we are ok to get rid of single quotes then we can do.

Reply:
Yeah, it's fine. I'm more concerned about the error message and the values in `VALID_ACTION_SOURCES` drifting apart.